### PR TITLE
Fix compilation errors

### PR DESCRIPTION
**Pull request by magicus**
First step. Now it compiles, at least. :)

### CODE REVIEW

**Comment by 7kasper** [comment]
Should all be readfloatLE

File: src/protocolsupportpocketstuff/hacks/holograms/HologramsPacketListener.java [hunk 1]
```diff
@@ -96,9 +96,9 @@ public void onRawPacketSending(RawPacketEvent event) {
 			if (!cachedArmorStands.containsKey(entityId))
 				return;
 
-			float x = MiscSerializer.readLFloat(data);
-			float y = MiscSerializer.readLFloat(data) + (float) Y_OFFSET;
-			float z = MiscSerializer.readLFloat(data);
+			float x = data.readFloat();
```
Comment (7kasper) [inline src/protocolsupportpocketstuff/hacks/holograms/HologramsPacketListener.java]: data.readFloatLE()

File: src/protocolsupportpocketstuff/packet/play/PlayerMovePacket.java [hunk 2]
```diff
@@ -38,12 +38,12 @@ public int getPacketId() {
 	@Override
 	public void toData(Connection connection, ByteBuf serializer) {
 		VarNumberSerializer.writeSVarLong(serializer, entityId);
-		MiscSerializer.writeLFloat(serializer, x);
-		MiscSerializer.writeLFloat(serializer, y);
-		MiscSerializer.writeLFloat(serializer, z);
-		MiscSerializer.writeLFloat(serializer, pitch);
-		MiscSerializer.writeLFloat(serializer, headYaw);
-		MiscSerializer.writeLFloat(serializer, yaw); //head yaw actually
+		serializer.writeFloat(x);
```
Comment (7kasper) [inline src/protocolsupportpocketstuff/packet/play/PlayerMovePacket.java]: Also all write should be writeFloatLE

**Comment by magicus** [comment]
Updated fix to use proper LE version, and to update to a new version of Spigot which was required. Also removed uuidSkinCache-related dead code.

**Pull request merged**